tests/Feature/AuthorPageTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use App\Authors\Author;
+use App\Authors\AuthorRepository;
+use App\Gists\Gistlog;
+use Carbon\Carbon;
+use Mockery\MockInterface;
+
+class AuthorPageTest extends TestCase
+{
+    private $faker;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->faker = Faker\Factory::create();
```

Comment:
Would you mind importing this at the top? You can alias `Faker\Factory` as `FakerFactory` if you want, for clarity